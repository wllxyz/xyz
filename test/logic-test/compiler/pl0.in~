$const a=365,b=2
#var c,d
@procedure f(p1,p2);
{ 
	p1:=p1+p2 
};
{
	$if 5>a 
	$then 
		c:=5
	$fi;
	$write(5) 
}.
