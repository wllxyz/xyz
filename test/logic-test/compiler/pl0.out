$const a=365,b=2 #var c,d @procedure f(p1,p2); { 
	p1:=p1+p2 
	}; {
	$if 5>5 $then $if 2>2 $then c:=5 $fi $fi;
	$write(5)
	}.
